src/components/InventoryTable/EntityTableToolbar.js

```diff
@@ -357,7 +357,7 @@ EntityTableToolbar.propTypes = {
     showTags: PropTypes.bool,
     getTags: PropTypes.func,
     hasAccess: PropTypes.bool,
-    filterConfig: PrimaryToolbar.propTypes.filterConfig,
+    filterConfig: PropTypes.object,
```

Comment:
We can make use of the new TS definitions once we start converting the Inventory itself to TS.